prime/src/components/borrow/actions/UniswapAddLiquidityActionCard.tsx

```diff
@@ -265,27 +266,20 @@ export default function UniswapAddLiquidityActionCard(props: ActionCardProps) {
   let max1 = accountState.assets.token1Raw;
   // If token1 is selected, we need to swap the max amounts
   if (!isToken0Selected) [max0, max1] = [max1, max0];
-  const maxString0 = Math.max(0, max0 - 1e-6).toFixed(6);
-  const maxString1 = Math.max(0, max1 - 1e-6).toFixed(6);
+  const maxString0 = max0.toString(GNFormat.DECIMAL);
+  const maxString1 = max1.toString(GNFormat.DECIMAL);
```

Comment:
I believe the max is not needed here. It appears to have been used when `max0` or `max1` is 0, meaning the result would be negative when subtracting the `1e-6` from it. Since we no longer subtract `1e-6`, this should no longer be a problem.

Reply:
Sounds good. I was worried that sometimes `max0`/`max1` itself may be negative. If that's the case we'll find out rather quickly.